pkg/provisioningv2/rke2/planner/planner_test.go

```diff
@@ -245,3 +247,67 @@ func Test_anyRoleWithoutWindows(t *testing.T) {
 		})
 	}
 }
+
+func Test_getInstallerImage(t *testing.T) {
```

Comment:
Another possible test case is ensuring the proper hierarchy between the global config and the machine config is maintained. We expect the global-config registry value to take precedence over the machine-config value if both are defined